src/screens/OthersScreen.tsx

```diff
@@ -196,64 +198,35 @@ const OthersScreen: React.FC<Props> = ({ navigation }) => {
             </AccessibleTouchableOpacity>
           </View>
         </View>
-
-        <View style={styles.withPaddingHorizontal}>
-          <View style={styles.textWrapper}>
-            <Text style={[styles.text, { color: colors.text }]}>
-              {t('about:followUs')}
-            </Text>
-          </View>
-          <View style={styles.socialRow}>
-            <AccessibleTouchableOpacity
-              accessibilityLabel="Twitter"
-              accessibilityRole="link"
-              accessibilityHint={`${t('open')} Twitter`}
-              onPress={() =>
-                handleSocialPress(
-                  'twitter://user?screen_name=meteorologit',
-                  'https://twitter.com/meteorologit'
-                )
-              }
-              style={styles.withMarginRight}>
-              <Icon
-                name={dark ? 'social-twitter-dark' : 'social-twitter-light'}
-                style={{ color: colors.text }}
-              />
-            </AccessibleTouchableOpacity>
-            <AccessibleTouchableOpacity
-              accessibilityLabel="Instagram"
-              accessibilityRole="link"
-              accessibilityHint={`${t('open')} Instagram`}
-              onPress={() =>
-                handleSocialPress(
-                  'instagram://user?username=ilmatieteenlaitos',
-                  'https://www.instagram.com/ilmatieteenlaitos/'
-                )
-              }
-              style={styles.withMarginRight}>
-              <Icon
-                name={dark ? 'social-instagram-dark' : 'social-instagram-light'}
-                style={{ color: colors.text }}
-              />
-            </AccessibleTouchableOpacity>
-            <AccessibleTouchableOpacity
-              accessibilityLabel="YouTube"
-              accessibilityRole="link"
-              accessibilityHint={`${t('open')} YouTube`}
-              onPress={() =>
-                handleSocialPress(
-                  'youtube://ilmatieteenlaitos',
-                  'https://www.youtube.com/user/ilmatieteenlaitos'
-                )
-              }
-              style={styles.withMarginRight}>
-              <Icon
-                name={dark ? 'social-youtube-dark' : 'social-youtube-light'}
-                style={{ color: colors.text }}
-              />
-            </AccessibleTouchableOpacity>
+        {socialMediaLinks.length > 0 && (
+          <View style={styles.withPaddingHorizontal}>
+            <View style={styles.textWrapper}>
+              <Text style={[styles.text, { color: colors.text }]}>
+                {t('about:followUs')}
+              </Text>
+            </View>
+            <View style={styles.socialRow}>
+              {socialMediaLinks.map((socialMediaLink) => (
+                <AccessibleTouchableOpacity
```

Comment:
Add `key={socialMediaLink.name}` to fix error: Warning: Each child in a list should have a unique "key" prop.